frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -8,7 +8,7 @@ export const listPage = {
     cy.byLegacyTestID('namespace-bar-dropdown').should('not.exist'),
   clickCreateYAMLdropdownButton: () => {
     return cy
-      .byLegacyTestID('dropdown-button')
+      .get('#item-create')
```

Comment:
Hi, I know I said this was ok, but I just found where you can add a `data-test="item-create"` [here](https://github.com/openshift/console/blob/1c9b925b8d722e9781a60c11fe39258d90bf916a/frontend/public/components/factory/list-page.jsx#L220).
Then you can use `.byTestID('item-create')...`.  
Thanks

Reply:
Can you pls also remove the `return` in `return cy..`?  -thanks

Reply:
Okay, updated the code.